presidio-anonymizer/presidio_anonymizer/entities/engine/anonymizer_result.py

```diff
@@ -0,0 +1,57 @@
+import logging
+from typing import Dict
+
+from presidio_anonymizer.entities.engine import PIIEntity
+from presidio_anonymizer.entities.engine.result import OperatorResult
+
+
+class AnonymizerResult(PIIEntity):
+    """Information about the encrypted entity."""
+
+    def __init__(self, start: int,
+                 end: int, entity_type: str) -> 'AnonymizerResult':
+        """
+        Have the information about the entity we already encrypted and want to decrypt.
+
+        :param key: the key used to anonymize/encrypt the text.
+        :param start: start index in the anonymized text.
+        :param end: end index in the anonymized text.
+        """
+        PIIEntity.__init__(self, start, end, entity_type)
+        self.logger = logging.getLogger("presidio-anonymizer")
+
+    def __gt__(self, other) -> bool:
+        """Check result is greater by the text index start location."""
+        return self.start > other.start
+
+    @classmethod
+    def from_json(cls, json: Dict) -> 'AnonymizerResult':
```

Comment:
can be moved to base class?

Reply:
Not really, part of it can, but you need a specific implementation, if you move this to the base, you will not know which instance you are creating.